BML_TUX_Project/Packages/bmlTUX/Scripts/Managers/SessionLogger.cs

```diff
@@ -37,8 +37,8 @@ public static class SessionLogger {
         public static void LogComplete(Session session) {
 
             //ensure log exists
-            if (File.Exists(FileLocationSettings.SessionLogFilePath)) {
-                using (StreamWriter streamWriter = new StreamWriter(FileLocationSettings.SessionLogFilePath, true)) {
+            if (File.Exists(session.saveFilePath)) {
+                using (StreamWriter streamWriter = new StreamWriter(session.saveFilePath, true)) {
```

Comment:
This is proof that your updates are warranted. The names make way more sense now. nice.